docsrv/fetcher_test.go

```diff
@@ -0,0 +1,28 @@
+package docsrv
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	testOwner   = "erizocosmico"
```

Comment:
are we testing against your org?
I thing that Retrival guys have something to solve that...